### PR TITLE
PT-4258: Get Jira keys for latest release

### PR DESCRIPTION
**Pull request by mvktsk**
Added function to the action to get Jira keys for the latest release (results of  vc-build QuickRelease command)

### CODE REVIEW

**Review by krankenbro — APPROVED** [review]

File: .github/workflows/test-jira-build.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@ name: test-jira-build
 
 on:
   workflow_dispatch:
+  push:
+    branches:
+      [PT-4258]
```
Comment (Vectorfield4) [inline .github/workflows/test-jira-build.yml:7]: Temporary changes?

File: get-jira-keys/src/index.ts [hunk 2]
```diff
@@ -1,27 +1,47 @@
 import * as github from '@actions/github'
 import * as core from '@actions/core'
 
-const regex = /((([A-Z]+)|([0-9]+))+-\d+)/g;
+
+const githubToken = process.env['GITHUB_TOKEN'];
+const ref = github.context.ref;
+const payload = github.context.payload;
+
+const keyRgx = /((([A-Z]+)|([0-9]+))+-\d+)/g;
```
Comment (Vectorfield4) [inline get-jira-keys/src/index.ts:9]: image
No reason for declaring variables in the global scope. It decreases readability. 

**For example:**
Move `keyRgx` to the `mathchKeys` function and change the name of `keyRgx` for something more objective (`jiraKeyRegex`).

File: get-jira-keys/src/index.ts [hunk 3]
```diff
@@ -46,33 +68,102 @@ async function getJiraKeysFromPr() {
...
+            owner: payload.repository.owner.login,
+            repo: payload.repository.name,
+            sha: ref,
+            since: sinceIsoString,
+            per_page: 100,
+        });
+
+        commitsArr = octokitResult['data'];
+        
+        if (!commitsArr) {
```
Comment (Vectorfield4) [inline get-jira-keys/src/index.ts:118]: Add comment why.

File: get-jira-keys/src/index.ts [hunk 4]
```diff
@@ -46,33 +68,102 @@ async function getJiraKeysFromPr() {
...
-async function run(): Promise<void> {
+async function getJiraKeysFromRelease() {
+    try {
+        console.log("Get Jira keys for latest release");
+
+        //Commits number contains 'release/x.x.x' message
+        const releaseMsgNum = 1; 
+
+        let resultArr: any = [];
+        let commitsArr: any = [];
```
Comment (Vectorfield4) [inline get-jira-keys/src/index.ts:97]: 1. Do not put data type literals inside the variable name (`resultArr` -> `jiraKeys`, `commitsArr` -> `commits`); 
2. Its good if you use strong types.

**Review by Vectorfield4 — CHANGES_REQUESTED** [review]

File: .github/workflows/test-jira-build.yml (hunk 1 above)
Comment (mvktsk) [reply .github/workflows/test-jira-build.yml:7]: This workflow for tests only

File: get-jira-keys/src/index.ts (hunk 2 above)
Comment (mvktsk) [reply get-jira-keys/src/index.ts]: fixed

File: get-jira-keys/src/index.ts (hunk 3 above)
Comment (mvktsk) [reply get-jira-keys/src/index.ts]: fixed

File: get-jira-keys/src/index.ts (hunk 4 above)
Comment (mvktsk) [reply get-jira-keys/src/index.ts]: fixed

**Review by Vectorfield4 — APPROVED** [review]

**Pull request merged**